### PR TITLE
Bumps k + k-a versions to use monasca-thresh and grafana fixes

### PR DESCRIPTION
**Pull request by Wasaac**

### CODE REVIEW

File: etc/kayobe/kolla.yml [hunk 1]
```diff
@@ -37,7 +37,7 @@ kolla_ansible_source_url: https://github.com/stackhpc/kolla-ansible.git
 
 # Version (branch, tag, etc.) of Kolla Ansible source code repository if type
 # is 'source'.
-kolla_ansible_source_version: refs/tags/stackhpc/7.1.1.2-rc
+kolla_ansible_source_version: refs/tags/stackhpc/7.1.1.3
```
Comment (dougszumski) [inline etc/kayobe/kolla.yml]: nit: It looks like we never had a 7.1.1.2 release - can we use that?

Comment (Wasaac) [reply etc/kayobe/kolla.yml]: Can do!

File: etc/kayobe/kolla/globals.yml [hunk 2]
```diff
@@ -48,14 +48,14 @@ keepalived_tag: 7.0.2.1-1
 keystone_tag: 7.0.2.1-1
 kibana_tag: 7.1.0.1
 kolla_toolbox_tag: 7.0.2.1-1
-logstash_tag: 7.1.0.1
```
Comment (Wasaac) [inline etc/kayobe/kolla/globals.yml]: Confession: I'm not sure how some of the image tags in globals have gone backwards.

Comment (dougszumski) [reply etc/kayobe/kolla/globals.yml]: There should be no need to change this one - we haven't patched the image.

Comment (dougszumski) [reply etc/kayobe/kolla/globals.yml]: Looks good now apart from this

Comment (Wasaac) [reply etc/kayobe/kolla/globals.yml]: Once again you're correct. For a moment I thought I'd put my monasca-log-transform fixes in kolla - they use the logstash image and would require updating the tag. In fact, those fixes are in kolla-ansible.

**Comment by Wasaac** [comment]
Realised that we'd also skipped the (edit:) 7.0.2.6 kolla tag - decremented to not waste that tag. Also realised that monasca-thresh changes in kolla necessitate updating the tag for that image.

File: etc/kayobe/kolla.yml [hunk 3]
```diff
@@ -15,7 +15,7 @@ kolla_source_url: https://github.com/stackhpc/kolla.git
 
 # Version (branch, tag, etc.) of Kolla source code repository if type is
 # 'source'.
-kolla_source_version: refs/tags/stackhpc/7.0.2.6-rc
+kolla_source_version: refs/tags/stackhpc/7.0.2.6
```
Comment (dougszumski) [inline etc/kayobe/kolla.yml:18]: This doesn't exist right now - are you in the process of creating it?

Comment (Wasaac) [reply etc/kayobe/kolla.yml:18]: Indeed. It exists now.

File: etc/kayobe/kolla/globals.yml [hunk 4]
```diff
@@ -55,10 +55,10 @@ mariadb_tag: 7.0.2.1-1
 memcached_tag: 7.0.2.1-1
 monasca_tag: 7.1.0.1
 monasca_api_tag: 7.0.2.3-1
-monasca_grafana_tag: 7.0.2.5-1
+monasca_grafana_tag: 7.0.2.6-2
 monasca_log_api_tag: 7.0.2.2-1
 monasca_notification_tag: 7.0.2.2-1
-monasca_thresh_tag: 7.0.2.6-1
+monasca_thresh_tag: 7.0.2.6-2
```
Comment (dougszumski) [inline etc/kayobe/kolla/globals.yml]: Hasn't the fix for this already been applied? We are running `7.0.2.6-1` for thresholder on both prod and alt-1.

Comment (Wasaac) [reply etc/kayobe/kolla/globals.yml]: Right you are again! Removed the offending line.

**Review by dougszumski — APPROVED** [review]
Looks good now.

It's a bit quirky because I think when the thresholder fix was rolled out, the image was tagged with `7.0.2.6-1` but the branch didn't actually exist (it was `7.0.2.6-1-rc`).

**Pull request merged**